test/unit/FilesExistTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace SendGrid\Test;
+
+class FilesExistTest extends \PHPUnit_Framework_TestCase
+{
+    public function testFileArePresentInRepo()
+    {
+        $this->assertFileExists('./.gitignore');
```

Comment:
```
There was 1 failure:
1) SendGrid\Test\FilesExistTest::testFileArePresentInRepo
Failed asserting that file "./.gitignore" exists.
/home/travis/build/sendgrid/php-http-client/test/unit/FilesExistTest.php:9
```

I am wondering if PHP is not checking the root directory, but in fact, checking the test file's directory? 
Can you please try outputting the directory here to test and fixing the file paths accordingly?

Reply:
yup, will try to do it asap

Reply:
Thank you!